### DISCUSSION TITLE
Platform

### DISCUSSION THREAD

**Discussion by Zeynex** [phst-randomizer/ph-randomizer#246] [Q&A] [answered]
Is the intended platform pc[emulation]? ds or wiiu? Asking for how the controls gonna be ^^

**Comment by mike8699** [comment, marked as answer]
The final output of the randomizer will be a patched .nds file, which will be runnable on any of the platforms you listed. As for controls, at this time the plan is for the randomizer to fully support both the original touch-based controls as well as the [dpad patch](https://www.romhacking.net/hacks/2248/).
